### ISSUE TITLE
fix: Skeleton object element does not have accessible name

### ISSUE THREAD

**Issue by break-stuff** [microsoft/fast#5827]
# 🐛 Bug Report

We are seeing accessibility errors with our audit tools. 

## 💻 Repro or Code Sample

The issue was discovered when running a FastPass audit with [Accessibility Insights For Web](https://accessibilityinsights.io/) browser extension.

https://jsfiddle.net/80vL3zu9/

## 💁 Possible Solution

Based on the report, it looks like the `object` element requires a `title`.

## 🌍 Your Environment

* OS & Device: [e.g. MacOS, iOS, Windows, Linux] on [iPhone 7, PC]
* Browser [e.g. Microsoft Edge, Google Chrome, Apple Safari, Mozilla FireFox]
* Version [e.g. 1.8.0]

**Comment by chrisdholt** [comment]
Thanks for the issue @break-stuff - I just ran Lighthouse and wasn't able to repro there, can you share a screenshot of the exact error using A11y insights for web? That'll help us focus effort.

**Comment by break-stuff** [comment]
Sure! Here is the error I am getting from Accessibility Insights for Web

![MicrosoftTeams-image](https://user-images.githubusercontent.com/31320098/163250951-f207f876-c520-4c1c-b2c8-f052248e3c61.png)

**Comment by chrisdholt** [comment]
After digging in here, I think this can be resolved with `role="presentation"`. In the case of skeleton, the host element itself should likely have an aria-label (or aria-labelledby) to describe it's purpose on the page. Will get a PR out soon for this.
